src/features/copilotTools.ts

```diff
@@ -139,12 +139,8 @@ export class GetEnvironmentInfoTool implements LanguageModelTool<IResourceRefere
 
 function BuildEnvironmentInfoContent(envInfo: EnvironmentInfo): LanguageModelTextPart {
     // Create a formatted string that looks like JSON but preserves comments
-    let envTypeDescriptor: string = `This environment is managed by ${envInfo.type} environment manager. Use the install tool to install packages into this environment.`;
+    const envTypeDescriptor: string = `This environment is managed by ${envInfo.type} environment manager. Use the install tool to install packages into this environment.`;
 
-    if (envInfo.type === 'system') {
-        envTypeDescriptor =
-            'System pythons are pythons that ship with the OS or are installed globally. These python installs may be used by the OS for running services and core functionality. Confirm with the user before installing packages into this environment, as it can lead to issues with any services on the OS.';
-    }
```

Comment:
Given the fact that we now have `confirmation` prompts in the install tool, these messages are not required.

Reply:
@eleanorjboyd @karthiknadig /cc